docs/devnet.md

```diff
@@ -0,0 +1,135 @@
+# Devnet Environment
+
+The devnet environment is constructed atop the OP stack devnet, also known as "bedrock". Essentially, this environment runs `op-geth` and `geth` node in development mode, excluding the PoS client, and without finality. This is why the `--devnet` flag is necessary. This flag allows for the acceptance of any block as finalized. Otherwise, the Magi would operate exclusively with finalized blocks.
```

Comment:
This kind of implies that the devnet is called bedrock, rather than the op stack. Can we make this clearer?

Reply:
You are right, my eyes glazed over 👍 Fix https://github.com/a16z/magi/pull/171/commits/9f420c9f6055939afe5a3e11ade2190336027973